src/networks/index.ts

```diff
@@ -123,6 +123,30 @@ export const networksByChainId: INetworkByChainId = {
     isMainnet: false,
     enabled: false,
   },
+  [ChainId.Shasta]: {
+    name: 'Shasta',
+    chainId: ChainId.Shasta,
+    payrollAddress: 'TV7Ui3s45msWixKG8g8nAFzqoWXQZvmY3B',
+    routerAddress: '',
+    wCurrencyAddress: '',
+    isSwapV2: true,
+    addressExplorerUrl: 'https://shasta.tronscan.org/#/address/',
+    txExplorerUrl: 'https://shasta.tronscan.org/#/transaction/',
+    isMainnet: false,
+    enabled: true,
+  },
+  [ChainId.Tron]: {
+    name: 'Tron',
+    chainId: ChainId.Tron,
+    payrollAddress: '',
+    routerAddress: '',
+    wCurrencyAddress: '',
```

Comment:
this is the wtrx contract `TNUC9Qb1rRpS5CbWLmNMxXBjyFoydXjWFR`

Reply:
Done!